contributions/catalog/catalog_rest_service.py

```diff
@@ -21,31 +21,39 @@
     logging.basicConfig(datefmt='%Y-%m-%dT%H:%M:%S', format=log_format, level=logging.DEBUG)
 else:
     logging.basicConfig(datefmt='%Y-%m-%dT%H:%M:%S', format=log_format, level=logging.INFO)
-if cfg and cfg.URL_PREFIX:
-    prefix = cfg.URL_PREFIX
-    staticpath = prefix + '/static'
+
+URL_PREFIX = os.getenv("URL_PREFIX", "")
+if URL_PREFIX:
+    staticpath = URL_PREFIX + '/static'
 else:
     staticpath = '/static'
 
 template_dir = os.path.join(os.path.abspath('webapps'), 'templates')
 static_dir = os.path.join(os.path.abspath('webapps'), 'static')
 app = Flask(__name__, instance_relative_config=True, static_url_path=staticpath, static_folder=static_dir,
             template_folder=template_dir)
-app.config.from_object(cfg)
 
 init_app(app)
 app.register_blueprint(contribute_bp)
 
+# read configs variable from .env file
```

Comment:
Based on the approach mentioned here https://github.com/rokwire/rokwire-building-blocks-api/pull/615/files#r567139398, I think you should be able to continue to import config variables directly into flask application from object like previously `app.config.from_object(cfg)`. This way, you don't have to import these config variables separately like below lines below 40. Hopefully, this will make the code simpler to read. Please look into this. Thanks.

Reply:
I just updated it